src/utils/communitySbt/helpers.ts

```diff
@@ -65,4 +66,23 @@ export function get2MRefereeBenchmark(subgraphUrl: string) : number {
 export function get100KRefereeBenchmark(subgraphUrl: string) : number {
     return subgraphUrl.includes("goerli") || subgraphUrl.includes("testnet") ? 
         GOERLI_ONE_HUNDRED_THOUSAND : MAINNET_ONE_HUNDRED_THOUSAND;
-}
\ No newline at end of file
+}
+
+export async function geckoEthToUsd(apiKey: string) : Promise<number> {
+    if (!process.env.REACT_APP_COINGECKO_API_KEY) {
```

Comment:
this should be passed to this function as param, and passed to the SDK from the UI

Reply:
I removed this ad the api key can't be undefined a889530